src/scss/05-useful/_display.scss

```diff
@@ -5,11 +5,31 @@
 
 ///
 .display {
+	&-none {
+		display: none;
+	}
+
 	&-block {
 		display: block;
 	}
 
 	&-contents {
 		display: contents;
 	}
+
+	&-inline {
+		display: inline;
+	}
+
+	&-inline-block {
+		display: inline-block;
+	}
+
+	&-inline-flex {
+		display: inline-flex;
+	}
+
+	&-inline-grid {
```

Comment:
I would call this one just `&-grid` and remove the inline

Reply:
@df-gss can you please review this before we merge the code? 
Thank you

Reply:
I changed it. It should indeed be called 'display-grid'.